docs/contributing/coding-standards/css.md

```diff
@@ -453,13 +447,13 @@ Bad:
 Good:
 ```
 .selector {
-  font-size: 0.5em;
+  font-size: .5em;
 }
 ```
 
 ### Remove trailing whitespace
 
-More write up on [supported rules](https://github.com/sasstools/sass-lint/tree/master/docs/rules).
+More write up on [supported rules](https://github.com/stylelint/stylelint/blob/master/docs/user-guide/configure.md#rules).
```

Comment:
I'm not entirely sure what the need for this link is, but wondering if https://stylelint.io/user-guide/rules/about or https://stylelint.io/user-guide/rules/list might be a better place to link to – the current URL seems to be more about configuring rules or adding entirely new ones.